icu4c/source/i18n/unitconverter.cpp

```diff
@@ -290,6 +301,8 @@ void addFactorElement(Factor &factor, StringPiece elementStr, SigNum sigNum, UEr
 
 /*
  * Extracts `Factor` from a complete string factor. e.g. "ft2m^3*1007/cup2m3*3"
+ *
+ * TODO: warning: unused parameter 'status' [-Wunused-parameter]
```

Comment:
Delete this TODO, I'm expecting the warning won't be given anymore? (You're using "status" now).

I think this was again me touching this file, and not trusting I (we?) will notice the build warning later. I should be more trusting! :-)

Reply:
Yes, it's used now. Hence I'm saying delete the TODO.

(I added this TODO when I touched younies' branch, because I find it hard to just ignore a warning. :-) I then habitually make a note of it in the codebase to not forget to take care of it before sending a PR.)